### PR TITLE
Feature/replace ace editor

### PR DESCRIPTION
**Pull request by Sisha3342**

### CODE REVIEW

**Review by yadamskaya — COMMENTED** [review]
One more thing with padding, I think it will be better to remove it
image

File: src/plugins/editor/jar/jar-editor.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  /** Normalize markup indents */
+  private normalize(markup: string): string {
+    return Prism.plugins.NormalizeWhitespace.normalize(markup);
+  }
+
+  /** Handle editor's content change */
+  @bind
+  private _onChange() {
+    this.dispatchEvent(new CustomEvent('editor-change'));
```
Comment (nattallius) [inline src/plugins/editor/jar/jar-editor.ts:55]: I'd better add prefix `uip` to event name as we do for all our events and maybe change it a little bit. As a reference, how every `setting` send change event to the main `settings` component: `uip:change`

Or if we consider these changes as a third- party library change mybe we shoul leave it without prefix....

**Review by yadamskaya — APPROVED** [review]

**Pull request merged**